frontend/src/services/fetchData.ts

```diff
@@ -9,33 +9,43 @@ import {
 } from '@/services/ApiServiceFactory';
 import FamLoginUserState from '@/store/FamLoginUserState';
 import { selectedApplicationId } from '@/store/ApplicationState';
+import { isNewAccess } from './utils';
 
 // --- Fetching data (from backend)
 
 export const fetchUserRoleAssignments = async (
-    applicationId: number | undefined
+    applicationId: number | undefined,
+    newUserAccessIds: string = ''
```

Comment:
all the fetch method will have a default value for query parameters, so inside method, when we the split method, it will return an empty string if no query parameter value is provided